worker.py

```diff
@@ -809,6 +829,31 @@ def bag_full(self):
         self.logger.info("Removed %d items", removed)
         self.error_code = '!'
 
+    async def incubate_eggs(self):
+        # copy the list, as self.call could modify it as it updates the inventory
+        incubators = list(self.unused_incubators)
+        for egg in sorted(self.eggs.values(), key=lambda x: x.get('egg_km_walked_target')):
+            if not egg.get('egg_incubator_id'):
+                continue
+
+            if not incubators:
+                break
+
+            inc = incubators.pop()
+            if ((inc.get('item_id', 902) == 901 or egg.get('egg_km_walked_target', 0) > 9)
+                    and not self.eggs[egg.get('id')].get('egg_incubator_id', False)):
+                request = self.api.create_request()
+                request.use_item_egg_incubator(item_id=inc.get('id'), pokemon_id=egg.get('id'))
+                responses = await self.call(request, action=5)
+
+                ret = responses.get('USE_ITEM_EGG_INCUBATOR', {}).get('result', 0)
+                if ret == 4:
+                    self.logger.warning("Tried reapply incubator to same egg, this shouldn't happen")
```

Comment:
Could you change it to `'Failed to use incubator because it was already in use.'`